### ISSUE TITLE
Steam games won't run [Bug]

### ISSUE THREAD

**Issue by neptalactone** [owl4ce/dotfiles#29]
So i encounter this issue, but i'm not sure if it's a bug or not. I noticed that i can't play any steam games in this environment, games seems like encounter some unexpected behavior and crash. I assume this happen because in your configuration, application that going full screen, forced not to be really full screen (there's a little gap).

This behavior didn't occur when i tried to play steam games on full desktop environment like KDE. Games run well and didn't encounter any crash since. 

I'm using this on my laptop from fresh installed Arch Linux and proceed with your installation guide for the themes and all.

Is there any way to fix this problem? I don't want to log in to KDE everytime i want to play a game. Thanks!

**Comment by owl4ce** [comment]
Screenshot needed :)

**Comment by nullxception** [comment]
I saw gtk-nocsd used on this setup, maybe that was the culprit ?
that thing known to breaks proton

**Comment by neptalactone** [comment]
> Screenshot needed :)

Here is the screenshot.

![2020-12-20-010303_1920x1080_scrot](https://user-images.githubusercontent.com/34985816/102687903-31b37600-4225-11eb-878c-fd3902d4b263.png)

I can't capture much. I tried, to throw the game to the terminal to see what error it shows with 'termite -c "%command%"' on the launch option. But it didn't show anything. I think, it have to do with the feature on this environment that let the maximize application appear to be no really "maximize" since it leaves a little gap.

**Comment by neptalactone** [comment]
> I saw gtk-nocsd used on this setup, maybe that was the culprit ?
> that thing known to breaks proton

is it a config file or a package? do you know how to disable it?

**Comment by neptalactone** [comment]
> > Screenshot needed :)
> 
> Here is the screenshot.
> 
> ![2020-12-20-010303_1920x1080_scrot](https://user-images.githubusercontent.com/34985816/102687903-31b37600-4225-11eb-878c-fd3902d4b263.png)
> 
> I can't capture much. I tried, to throw the game to the terminal to see what error it shows with "PROTON LOG=1 %command%" on the launch option. But it didn't show anything. I think, it have to do with the feature on this environment that let the maximize application appear to be no really "maximize" since it leaves a little gap.

**Comment by nullxception** [comment]
> > I saw gtk-nocsd used on this setup, maybe that was the culprit ?
> > that thing known to breaks proton
> 
> is it a config file or a package? do you know how to disable it?

disable it in https://github.com/owl4ce/dotfiles/blob/master/.config/openbox/environment#L7..L9

**Comment by neptalactone** [comment]
> > > I saw gtk-nocsd used on this setup, maybe that was the culprit ?
> > > that thing known to breaks proton
> > 
> > 
> > is it a config file or a package? do you know how to disable it?
> 
> delete those lines https://github.com/owl4ce/dotfiles/blob/master/.config/openbox/environment#L7..L9

It's working now! Thanks! 
Why it cause proton steam games to not launch? What is this setting for?

**Comment by nullxception** [comment]
> > > > I saw gtk-nocsd used on this setup, maybe that was the culprit ?
> > > > that thing known to breaks proton
> > > 
> > > 
> > > is it a config file or a package? do you know how to disable it?
> > 
> > 
> > delete those lines https://github.com/owl4ce/dotfiles/blob/master/.config/openbox/environment#L7..L9
> 
> It's working now! Thanks!
> Why it cause proton steam games to not launch? What is this setting for?

Glad it works ☺️

gtk-nocsd is an unofficial hacks to disable...well.. the client-side decoration,
and perhaps it causes some proton runtime that depends on gtk to breaks.

tldr; it's an unofficial stuff so not surprising if it breaks

**Comment by owl4ce** [comment]
**Hi! Thanks @nullxception for helping answer. Also @neptalactone for reporting this bug.**

Out of context but still about openbox/environment, I will only add a description of the problem for qt.
Since a long time ago I have uniformed the qt theme with gtk but it's for a dark theme that doesn't work maybe because the background is static for some apps. So that only the font changes color to be like this.

_https://wiki.archlinux.org/index.php/Qt#Configuration_of_Qt5_apps_under_environments_other_than_KDE_Plasma_

<br>

<p align="center"><b>Eyecandy</b></p>

![2020-12-20-051728_1366x768_scrot](https://user-images.githubusercontent.com/53987136/102700767-08c1ce00-4283-11eb-8eac-0332b650a71d.png)

<p align="center"><b>Mechanical</b></p>

![2020-12-20-051747_1366x768_scrot](https://user-images.githubusercontent.com/53987136/102700770-0b242800-4283-11eb-8c7d-4f4e1cdeb5b8.png)


The solution also turns off the line:
`export QT_QPA_PLATFORMTHEME=qt5ct`
